packages/developer-console-spa/src/components/ConfigureCouchDB.tsx

```diff
@@ -32,7 +32,8 @@ function ConfigureCouchDB ( props: any ) {
     return string.replace(/[-\\^$*+?.()|[\]{}]/g, "\\$&");
   };
   const regex = new RegExp( escapeString( searchTerm.trim() ), "i" );
-  const filteredDBs = app.database.databases.filter( ( db: string ) => regex.test( db ) );
+
+  const filteredDBs = app.database?.databases.filter( ( db: Database ) => regex.test( db.name ) );
```

Comment:
Do we really need regex here.

Reply:
@deshmukhmayur WDYT?

Reply:
I agree. Just simple `.includes` should suffice for this search.

Reply:
Done